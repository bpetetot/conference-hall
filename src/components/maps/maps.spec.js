/* eslint-env jest */
import snap from 'tests/snapshot'
import Maps from './index.js'

const snapshot = props => snap(Maps)({ ...props })

describe('components/maps', () => {
  it('should not render if no address', snapshot({}))
  it('should render if address', snapshot({ address: 'c' }))
})
