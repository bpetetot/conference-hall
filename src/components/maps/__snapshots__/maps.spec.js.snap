// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/maps should not render if no address 1`] = `null`;

exports[`components/maps should render if address 1`] = `
<iframe
  allowFullScreen={true}
  className="maps"
  frameBorder="0"
  src="https://www.google.com/maps/embed/v1/place?key=<API_KEY>&q=c"
  title="maps"
/>
`;
